### PR TITLE
Fix a typo in azurerm-ps-in-docker.md

### PR DESCRIPTION
**Pull request by alexandair**
Fix a typo.

### CODE REVIEW

**Comment by sptramer** [comment]
Thanks for this contribution. The rewording emphasizes that the sentence is awkward so there are some additional edits needed, to make it even clearer. I can either provide them myself or provide feedback if you would prefer to make them on your own.

**Comment by alexandair** [comment]
I don't understand what is awkward. "a set" is a singular noun, therefore we need "is".
Please, provide some additional edits if you think they are needed.

**Comment by sptramer** [comment]
@alexandair Your edit was correct, but it showed the other flaws in the sentence structure (overly technical and hard to parse on first scan) that should be smoothed over. I've gone ahead and made the changes that hopefully make the article much easier to understand.
